src/Workspaces/Remote/Core/ExternalAccess/UnitTesting/Api/UnitTestingRemoteHostOptionsAccessor.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingRemoteHostOptionsAccessor
+    {
+        public static Option<bool> OOPCoreClr => new(
```

Comment:
I am not sure if UT already has access to the `Option` type. Would we need a new wrapper type for that?

Reply:
Ah I think we don't have access to Option type iirc